### PR TITLE
Calc logic

### PR DESCRIPTION
**Pull request by alissonperes**
Create a new logic folder inside src and initialize two javascript files there: calculate.js and operate.js.

Implement the calculate.js module

Implement the operate.js module

Add calculate.js to the App component
    - Import the calculate function.

### CODE REVIEW

**Comment by alissonperes** [comment]
Added eslint-disable-line to the import of calculate in App.js since it's mandatory for this milestone
Added eslint-disable-line to import of big.js in operate.js because it was giving a linter error, however it was necessary for the import

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      result = numOne.minus(numTwo).toString();
+      break;
+    case '*':
+      // .times
+      result = numOne.times(numTwo).toString();
+      break;
+    case '÷':
+      // .div
+      result = numOne.div(numTwo).toString();
+      break;
```
Comment (shloch) [inline src/logic/operate.js:23]: Dont forget to handle division by ZERO on the next milestone

**Review by shloch — APPROVED** [review]
## @alissonperes , 

accomplished

File: src/logic/operate.js (hunk 1 above)
Comment (alissonperes) [reply src/logic/operate.js:23]: thanks for the tip :+1:

**Pull request merged**